### ISSUE TITLE
Trained model in GPU, Want to run the saved model in CPU. 

### ISSUE THREAD

**Issue by YakinRubaiat** [matterport/Mask_RCNN#1179]
I have a pre-trained model with Mask_RCNN on GPU and I am trying to run this saved model on CPU.
I already load the model in the interference model with GPU_COUNT = 1. But when I want to detect with 'model.detect' the following error occurs

`indices[590] = 16330 is not in [0, 16320)
[[{{node ROI_4/GatherV2_2}} = GatherV2[Taxis=DT_INT32, Tindices=DT_INT32, Tparams=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](ROI_4/strided_slice_6, ROI_4/strided_slice_7, roi_align_mask_4/concat/axis)]]
`

**Comment by cooleel** [comment]
I get the same error.
Do you know how to solve this problem?

**Comment by YakinRubaiat** [comment]
Try GPU_COUNT = 0

**Comment by cooleel** [comment]
@YakinRubaiat 
Thanks for the response, but I don't think this will work since the batch_size will be 0 in this case.
Did you solve this problem with `GPU_COUNT=0` ?

**Comment by Wei-1** [comment]
The problem is because of `RPN_ANCHOR_SCALES` and `BACKBONE_STRIDES` layer count mismatch.
https://github.com/matterport/Mask_RCNN/issues/1291#issuecomment-856142027
